### PR TITLE
zziplib, bump version, cleanup

### PR DESCRIPTION
**Pull request by Begasus**

### CODE REVIEW

**Comment by Begasus** [comment]
Test crashes here, build is OK, @alaviss ... since you did some work on previous fixes for the tests could you have a look here? :)

File: dev-libs/zziplib/zziplib-0.13.71.recipe [hunk 1]
```diff
@@ -5,34 +5,39 @@ HOMEPAGE="http://zziplib.sourceforge.net/"
...
-CHECKSUM_SHA256="846246d7cdeee405d8d21e2922c6e97f55f24ecbe3b6dcf5778073a88f120544"
+CHECKSUM_SHA256="2ee1e0fbbb78ec7cc46bde5b62857bc51f8d665dd265577cf93584344b8b9de2"
 PATCHES="zziplib-$portVersion.patchset"
 
 ARCHITECTURES="x86_gcc2 x86 x86_64"
 SECONDARY_ARCHITECTURES="x86"
 
-libVersion="13.0.69"
+libVersion="13.0.71"
 libVersionCompat="$libVersion compat >= 13"
```
Comment (alaviss) [inline dev-libs/zziplib/zziplib-0.13.71.recipe:17]:
```suggestion
libVersionCompat="$libVersion compat >= ${libVersion%%.*}"
```

File: dev-libs/zziplib/zziplib-0.13.71.recipe [hunk 2]
```diff
@@ -103,11 +113,11 @@ INSTALL()
 		libzzipfseeko \
 		libzzipmmapped \
 		libzzipwrap
-
 	fixPkgconfig
 
 	packageEntries devel \
-		$developDir
+		$developDir \
+		$dataDir
```
Comment (alaviss) [inline dev-libs/zziplib/zziplib-0.13.71.recipe:120]: We should move `man3` to the `devel` package too.

File: dev-libs/zziplib/zziplib-0.13.71.recipe (hunk 1 above)
Comment (Begasus) [reply dev-libs/zziplib/zziplib-0.13.71.recipe:17]: local copy already has 13.0.71, not sure why I didn't push that change :)

File: dev-libs/zziplib/zziplib-0.13.71.recipe (hunk 2 above)
Comment (Begasus) [reply dev-libs/zziplib/zziplib-0.13.71.recipe:120]: Also already in the local copy, should have pushed those changes

**Comment by Begasus** [comment]
Still get a crash :(
Terminal output: https://paste.ubuntu.com/p/Hd2nP7qNPY/
Debug report: https://paste.ubuntu.com/p/kvfk49yqRk/

**Comment by Begasus** [comment]
Crash is comming from this line ... https://github.com/gdraheim/zziplib/blob/c3de4fe291ab6756593a0ef6bad007ddd3953e00/test/zziptests.py#L3491

File: dev-libs/zziplib/patches/zziplib-0.13.71.patchset [hunk 3]
```diff
@@ -0,0 +1,189 @@
...
+diff --git a/test/zziptests.py b/test/zziptests.py
+index 1c2d19d..83b0345 100644
+--- a/test/zziptests.py
++++ b/test/zziptests.py
+@@ -3488,7 +3488,7 @@ class ZZipTest(unittest.TestCase):
+     self.assertFalse(run.returncode)
+     # list the ZIPfile
+     exe=self.bins("unzip-mem");
+-    run = shell("{chdir} {tmpdir} && ../{exe} -v {zipname}.zip".format(**locals()), returncodes = [0,-8])
++    run = shell("printf '%s\\n' ??*.* | {chdir} {tmpdir} && ../{exe} -v {zipname}.zip".format(**locals()), returncodes = [0,-8])
```
Comment (Begasus) [inline dev-libs/zziplib/patches/zziplib-0.13.71.patchset:158]: @alaviss this fixes the crash when running the tests, probably needs some love :)

**Comment by Begasus** [comment]
Switched to cmake build here, tests doesn't crash now (but the fix maybe could use some love :)
Test results:
Version 0.13.69:
```
Ran 207 tests in 257.227s
FAILED (failures=40, errors=12, skipped=1)
```
Version 0.13.71
```
Ran 209 tests in 258.824s
FAILED (failures=6, errors=1, skipped=7)
```
So a lot less failures in this version

**Comment by Begasus** [comment]
Failures are similar to the ones mentioned here: https://github.com/gdraheim/zziplib/issues/102
Added a PATCH() as used here: https://github.com/gentoo/gentoo/pull/17636/files#diff-001a4e7ab9355e3ae408fc78bfce8846R49

File: dev-libs/zziplib/patches/zziplib-0.13.71.patchset (hunk 3 above)
Comment (Begasus) [reply dev-libs/zziplib/patches/zziplib-0.13.71.patchset:158]: Failures are dealt with in the PATCH() function, this doesn't crash but still produces an error:
```
ERROR: test_65485_list_verbose_compressed_with_directory (__main__.ZZipTest)
 verbously list a zipfile containing directories
----------------------------------------------------------------------
Traceback (most recent call last):
  File "/sources/zziplib-0.13.71/test/zziptests.py", line 3348, in test_65485_list_verbose_compressed_with_directory
    run = shell("printf '%s\\n' ??*.* | {chdir} {tmpdir} && ../{exe} -v {zipname}.zip".format(**locals()), returncodes = [0,-8])
  File "/sources/zziplib-0.13.71/test/zziptests.py", line 102, in shell
    raise subprocess.CalledProcessError(run.returncode, sh_command, output)
subprocess.CalledProcessError: Command 'printf '%s\n' ??*.* | cd tmp.test_65485 && ../../bins/unzip-mem -v ZIPfile.zip' returned non-zero exit status 127.
```

Comment (Begasus) [reply dev-libs/zziplib/patches/zziplib-0.13.71.patchset:158]: Test disabled in the recipe for now, this can be fixed later

**Pull request merged**